sdk/include/psdk/winnls.h

```diff
@@ -116,42 +116,45 @@ extern "C" {
 #define LOCALE_SABBREVMONTHNAME13	0x100F
 #define LOCALE_SPOSITIVESIGN	80
 #define LOCALE_SNEGATIVESIGN	81
-#define LOCALE_SSCRIPTS         108 // FIXME
-#define LOCALE_IPOSSIGNPOSN	82
-#define LOCALE_INEGSIGNPOSN	83
-#define LOCALE_IPOSSYMPRECEDES	84
-#define LOCALE_IPOSSEPBYSPACE	85
-#define LOCALE_INEGSYMPRECEDES	86
-#define LOCALE_INEGSEPBYSPACE	87
-#define LOCALE_FONTSIGNATURE    88
-#define LOCALE_SISO639LANGNAME  89
-#define LOCALE_SISO3166CTRYNAME 90
-#define LOCALE_SNAME            92 // FIXME
-
-#if (WINVER >= 0x0600)
-#define LOCALE_SNAME            92
-#define LOCALE_SDURATION        93
-#define LOCALE_SKEYBOARDSTOINSTALL 94
-#define LOCALE_SSHORTESTDAYNAME1 96
-#define LOCALE_SSHORTESTDAYNAME2 97
-#define LOCALE_SSHORTESTDAYNAME3 98
-#define LOCALE_SSHORTESTDAYNAME4 99
-#define LOCALE_SSHORTESTDAYNAME5 100
-#define LOCALE_SSHORTESTDAYNAME6 101
-#define LOCALE_SSHORTESTDAYNAME7 102
-#define LOCALE_SISO639LANGNAME2  103
-#define LOCALE_SISO3166CTRYNAME2 104
-#define LOCALE_SNAN              105
-#define LOCALE_SPOSINFINITY      106
-#define LOCALE_SNEGINFINITY      107
-#define LOCALE_SSCRIPTS 108
-#define LOCALE_SPARENT  109
+#define LOCALE_IPOSSIGNPOSN         82
+#define LOCALE_INEGSIGNPOSN         83
+#define LOCALE_IPOSSYMPRECEDES      84
+#define LOCALE_IPOSSEPBYSPACE       85
+#define LOCALE_INEGSYMPRECEDES      86
+#define LOCALE_INEGSEPBYSPACE       87
+#define LOCALE_FONTSIGNATURE        88
+#define LOCALE_SISO639LANGNAME      89
+#define LOCALE_SISO3166CTRYNAME     90
+
+/* FIXME: This value should be in the guarded block below */
+#define LOCALE_SNAME                92
+
+#if (WINVER >= 0x0600) || (defined(__REACTOS__) && defined(_KERNEL32_))
```

Comment:
Why does this need a `__REACTOS__` guard?
(is it because we use, from the wine winnls code, these defines even if the kernel32 is compiled with WINVER == 0x0502 ?? 🤔 )

Reply:
Yes